src/utils/status-codes.js

```diff
@@ -33,6 +33,11 @@ export const STATUS_VARIATION_REJECTED = {
   description: 'Variation Rejected',
 }
 
+export const STATUS_NO_ACCESS = {
+  code: 1000,
+  description: 'No Access',
+}
+
```

Comment:
You will also need to include `STATUS_NO_ACCESS.description`, at the end of this file.